src/service/viewer-impl.js

```diff
@@ -109,7 +109,7 @@ export const VisibilityState = {
  * @export {!Array<!RegExp>}
  */
 export const TRUSTED_VIEWER_HOSTS = [
-  /^(.*\.)?(google)(\.com?)?(\.[a-z]{2})?$/,
+  /(^|\.)google\.(com?|[a-z]{2}|com?\.[a-z]{2})$/,
```

Comment:
now that we are changing it - could you document what is being matched here (couple of strings that pass / fail)